test/src/main/webapp/META-INF/context.xml

```diff
@@ -8,12 +8,12 @@
 		Configuration via driverClassName only works for the standard transaction manager.
 	-->
 
-
+	<!-- IbisProp table has a column named VALUE. VALUE is a keyword in h2 db. -->
 	<Resource
 		name="jdbc/ibis4test-h2"
 		factory="org.apache.naming.factory.BeanFactory"
 		type="org.h2.jdbcx.JdbcDataSource"
-		URL="jdbc:h2:file:./ibis4test;MODE=Oracle;DB_CLOSE_ON_EXIT=FALSE;AUTO_SERVER=TRUE;TRACE_LEVEL_FILE=0"
+		URL="jdbc:h2:file:./ibis4test;NON_KEYWORDS=VALUE;AUTO_SERVER=TRUE;DB_CLOSE_ON_EXIT=FALSE;TRACE_LEVEL_FILE=0"
```

Comment:
If there is no reason to move AUTO_SERVER=true to before DB_CLOSE_ON_EXIT, it can stay behind it. Then there are less 'unnecessary' changes, that will make it easier to comparisons later.